### DISCUSSION TITLE
Add a feature to revert back to how the studios in 2.0-3.0 looked

### DISCUSSION THREAD

**Discussion by ItsEnderium** [ScratchAddons/ScratchAddons#2922] [Ideas]
I don't like the new look, and that one is nicer and smaller.

**Comment by WorldLanguages** [comment]
I feel this will be highly requested, at least for now. We can also not do this and let people get used to it. Most people didn't like the new messages page at first, but nowadays the old one looks ugly.
Maybe an addon for having smaller project thumbnails in studios would be better.

**Comment by mxmou** [comment]
You already suggested something very similar: #2866

I think this is a good idea, but it should only change the layout (to be narrower, show the follow button in the top right, show more project thumbnails per row ...). I don't think it should change the font size.
